RealmSwift/Tests/ObjectAccessorTests.swift

```diff
@@ -89,17 +90,39 @@ class ObjectAccessorTests: TestCase {
         realm.write {
             let obj = SwiftAllIntSizesObject()
 
-            obj.int16 = v16
-            XCTAssertEqual(obj.int16, v16)
-            obj.int32 = v32
-            XCTAssertEqual(obj.int32, v32)
-            obj.int64 = v64
-            XCTAssertEqual(obj.int64, v64)
+            let testObject: Void -> Void = {
+                func setAndTestValue<O: Object, T: Equatable>(object: O, #value: T, #getter: (O) ->(T), #setter: (O, T) -> ()) {
```

Comment:
This function is pointless. The code using it is not shorter or simpler than just writing the tests directly.